packages/webapp/src/components/Task/TaskReadOnly/index.jsx

```diff
@@ -160,23 +163,14 @@ export default function PureTaskReadOnly({
         )}
       </div>
 
-      <Input
-        style={{ marginBottom: '40px' }}
-        type={'date'}
-        value={date}
-        label={dateLabel}
-        disabled
-      />
+      <div className={styles.assigneeContainer} style={{ marginBottom: '40px' }}>
+        <Input type={'date'} value={date} label={dateLabel} disabled />
+        {isCurrent && isAdmin && (
+          <BiPencil className={styles.pencil} onClick={(_) => setShowDueDateModal(true)} />
+        )}
+      </div>
```

Comment:
Can we change the spacing and indentation to be more like line 155~164?

Reply:
<img width="300" alt="Screen Shot 2022-05-09 at 4 28 52 PM" src="https://user-images.githubusercontent.com/104768598/167493116-92deb555-2205-4095-8b75-e10127e3efd4.png">